processing/src/main/java/org/apache/carbondata/processing/datatypes/PrimitiveDataType.java

```diff
@@ -94,6 +99,11 @@
 
   private CarbonDimension carbonDimension;
 
+  private Boolean isDictionary;
+
+  private FieldConverter fieldConverterForNoDictionary;
```

Comment:
FieldConverter is not required in this class, as caller will create for primitive type like existing flow..same way we can handle for no dictionary

Reply:
Removed